hydra-cluster/src/HydraNode.hs

```diff
@@ -401,8 +402,15 @@ withHydraNode' tracer chainConfig workDir hydraNodeId hydraSKey hydraVKeys allNo
     , i /= hydraNodeId
     ]
 
-withConnectionToNode :: forall a. Tracer IO HydraNodeLog -> Int -> (HydraClient -> IO a) -> IO a
-withConnectionToNode tracer hydraNodeId action = do
+withConnectionToNode :: forall a. Tracer IO HydraNodeLog -> Int -> Bool -> (HydraClient -> IO a) -> IO a
+withConnectionToNode tracer hydraNodeId =
+  withConnectionToNodeHost tracer hydraNodeId Host{hostname, port}
+ where
+  hostname = "127.0.0.1"
+  port = fromInteger $ 4_000 + toInteger hydraNodeId
```

Comment:
Could: just get rid of the `Int` variant and push the `4_000 + toInteger hydraNodeId` further upstream? or into a function `testApiHostFromNodeId :: Int -> Host`

Reply:
good idea.

lets address this in a subsequent refactor:

https://github.com/orgs/cardano-scaling/projects/6?pane=issue&itemId=77063030